src/app/views/activities-and-events/activities-and-events.component.ts

```diff
@@ -1,11 +1,36 @@
-import { Component } from '@angular/core';
+import { Component, ElementRef, ViewChild, OnInit } from '@angular/core';
+import { monsoonPageData } from '../../../data-entries/json/monsoon-schools';
 
 @Component({
   selector: 'app-activities-and-events',
   templateUrl: './activities-and-events.component.html',
   styleUrls: ['./activities-and-events.component.scss']
 })
-export class ActivitiesAndEventsComponent {
+export class ActivitiesAndEventsComponent implements OnInit {
+  years: string[] = [];
+  selectedYear: string = '';
+  eventData: any;
+  @ViewChild('scrollToTopButton') scrollToTopButton?: ElementRef;
```

Comment:
Not needed

Reply:
Removed

Reply:
Removing it, not helping the cause. May have to implement in other way altogether.